src/js/translation/Listing.js

```diff
@@ -63,6 +81,96 @@ Vue.component('translation-listing', {
+                        }
+
+                        let url = '/admin/translations/import';
+                        let formData = new FormData();
+
+                        formData.append('fileImport', this.file);
+                        formData.append('importLanguage', this.importLanguage);
+                        formData.append('onlyMissing', this.onlyMissing);
+
+                        axios.post(url, formData, {
+                            headers: {
+                                'Content-Type': 'multipart/form-data'
+                            }
+                        }).then(response => {
+                            if(response.data.hasOwnProperty('numberOfImportedTranslations') && response.data.hasOwnProperty('numberOfUpdatedTranslations')){
+                                this.currentstep = 3;
+                                this.numberOfSuccessfullyImportedLanguages = response.data.numberOfImportedTranslations;
+                                this.numberOfSuccessfullyUpdatedLanguages = response.data.numberOfUpdatedTranslations;
+                                this.loadData();
+                            } else {
+                                this.currentstep = 2;
+                                this.numberOfFoundTranslations = Object.keys(response.data).length;
+                                this.conflicts = response.data;
+
+                                for(let i = 0; i < this.conflicts.length; i++){
+                                    if(this.conflicts[i].hasOwnProperty('has_conflict')){
+                                        if(this.conflicts[i].has_conflict) {
+                                            this.numberOfTranslationsToReview++;
+                                        }
+                                    }
+                                }
+                            }
+                        }, error => {
+                            this.$notify({ type: 'error', title: 'Error!', text: 'An error has occured.'});
+                        });
+                    });
+            } else if(this.currentstep === 2){
+                return this.$validator.validateAll()
+                    .then(result => {
+                        if (!result) {
+                            this.$notify({ type: 'error', title: 'Error!', text: 'The form contains invalid fields.'});
+                            return false;
+                        }
+
+                        for(let i = 0; i < this.conflicts.length; i++){
+                            if(this.conflicts[i].hasOwnProperty('checkedCurrent')){
+                                if(this.conflicts[i].checkedCurrent) {
+                                    this.conflicts[i][this.importLanguage.toLowerCase()] = this.conflicts[i].current_value;
+                                }
+                            }
+                        }
+
+                        let url = '/admin/translations/import/conflicts';
```

Comment:
Do we need to hardcode the url here? I do not like this approach, should be provided as input.